Source/WebCore/Modules/fetch/FetchBody.cpp

```diff
@@ -250,7 +252,10 @@ RefPtr<FormData> FetchBody::bodyAsFormData() const
         return FormData::create(PAL::UTF8Encoding().encode(urlSearchParamsBody().toString(), PAL::UnencodableHandling::Entities));
     if (isBlob()) {
         auto body = FormData::create();
-        body->appendBlob(blobBody().url());
+        ASSERT(blobBody().scriptExecutionContext());
+        auto topOrigin = blobBody().scriptExecutionContext() ? blobBody().scriptExecutionContext()->topOrigin().data() : SecurityOrigin::createOpaque()->data();
```

Comment:
Seems like this should use SecurityOriginData::createOpaque() from your other PR.